src/classes/BDI_DataImportService.cls

```diff
@@ -72,6 +72,27 @@ global with sharing class BDI_DataImportService {
         return bdiMS;
     }
 
+    /*******************************************************************************************************
+    * @description constructs the soql String to run a Data Import process for list of DataImport ids.
+    * @param dataImportIds list of data import ids to include in query
+    * @return String the soql String
+    */
+    public static String strSoqlForDataImportProcess(List<Id> dataImportIds) {
+        String dataImportQuery;
+
+        // construct or SOQL for the batch
+        dataImportQuery = 'SELECT ' + String.join(listStrDataImportFields, ',');
+        dataImportQuery += ' FROM ' + UTIL_Namespace.StrTokenNSPrefix('DataImport__c');
```

Comment:
Can this line use the object token instead of introducing a string reference? 
`String.valueOf(DataImport__c)`

Reply:
@kenlewis-sfdo I think if we want to avoid the direct string reference, we might need to use the describe info of `DataImport__c`: `DataImport__c.SObjectType.getDescribe().localName`. The `valueOf` method accepts just the `Object` parameter

Reply:
I wasn't sure how much to keep this in parallel with the method directly below it, "strSoqlForBatchProcess()" - ideally we would refactor both to use the describe call with the localName or something, but that method is very foundational to BDI and gets called in so many contexts, I just want to be sure not to introduce a namespace bug somehow (and also want to keep these two methods in sync if possible, since they are similar in their function)

Reply:
I agree, I would go with the low-risk option given the widespread use of the method.

Reply:
In that case I would vote to refactor the other method.  IMO we should avoid introducing string references to known objects/fields and should be cleaning those up in old code where encountered.  Using tokens preserves referential integrity and bypasses the need to do any manual namespace checking.

Reply:
I will make that update - I certainly agree in principal that we should avoid string references; my only reservation is breaking the seal on code that's been running in production successfully for so long! But good point - let's get this up to date.